demo/theme/victory-axis-differential-styling-theme.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  axis: {
```

Comment:
I think I would much prefer if the themes ended up with new namespaces `dependentAxis` and `independentAxis` in addition to `axis` so that this change can be non-breaking, and so that props in addition to styles can be supplied to each axis independently.